pages/plans/[id]/share.js

```diff
@@ -0,0 +1,38 @@
+import { getPossessiveName } from 'lib/utils/name';
+import SharePlan from 'components/Feature/SharePlan';
+import { usePlan, requestPlan, HttpStatusError } from 'api';
+import { getToken } from 'lib/utils/token';
+
+const Share = ({ plan, planId, token }) => {
+  const { error, loading, sharePlan } = usePlan(planId, {
+    plan,
+    token
+  });
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <>
+      <h1>{getPossessiveName(plan.firstName, plan.lastName)} shared plan</h1>
```

Comment:
Wonder if we could refactor this in the future into a standalone like "PlanHeader" component, now that we are using it in two places?